### PR TITLE
feat: initial repository setup

### PR DESCRIPTION
**Pull request by Roro-Iyer**
Initial setup of repository completed

### CODE REVIEW

**Comment by esaldivar** [comment]
Hey take a look at this for me please.

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# RohanReactTypescriptTraining
\ No newline at end of file
+# RohanReactTypescriptTraining
+
```
Comment (esaldivar) [inline README.md:2]: Can we remove this blank space?

**Review by esaldivar — CHANGES_REQUESTED** [review]
Please see my notes on this file.

File: README.md (hunk 1 above)
Comment (Roro-Iyer) [reply README.md]: Done

**Review by esaldivar — APPROVED** [review]

**Pull request merged**